### PR TITLE
sci-biology/salmon: add salmon

### PR DESCRIPTION
**Pull request by mschubert**
> Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. Salmon achieves is accuracy and speed via a number of different innovations, including the use of lightweight alignments.

https://github.com/COMBINE-lab/salmon

### CODE REVIEW

File: sci-biology/salmon/salmon-0.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+HOMEPAGE="https://github.com/COMBINE-lab/salmon"
+SRC_URI="https://github.com/COMBINE-lab/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (jlec) [inline sci-biology/salmon/salmon-0.3.2.ebuild]: If empty, both can be dropped.

File: sci-biology/salmon/salmon-0.3.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+src_prepare() {
+	epatch "${FILESDIR}/${P}-no-boost-static.patch"
```
Comment (jlec) [inline sci-biology/salmon/salmon-0.3.2.ebuild]: Please use the PATCHES=().

**Comment by marbre** [comment]
Please add a short description to the patch and clean it up like described in http://dev.gentoo.org/~vapier/clean-patches

**Comment by mschubert** [comment]
Thank your for the hints, I'm learning something new with each PR :-)

Should be fixed now.

**Pull request merged**